composeApp/src/desktopMain/kotlin/edu/dyds/movies/data/local/LocalMoviesSource.kt

```diff
@@ -5,12 +5,11 @@ import edu.dyds.movies.domain.entity.Movie
 interface LocalMoviesSource {
     fun initializeMovieCache(popularMovies: List<Movie>): List<Movie>
```

Comment:
Aprovechemos para cambiar el nombre, y no debería retornar nada
```suggestion
    fun addMovies(movies: List<Movie>)
```